### PR TITLE
x11-themes/arc-kde: new package

### PR DESCRIPTION
**Pull request by gcarq**
This is a port of the popular GTK theme Arc for Plasma 5 desktop with a few additions and extras.

https://github.com/PapirusDevelopmentTeam/arc-kde

Signed-off-by: Michael Egger michael.egger@tsn.at
Package-Manager: Portage-2.3.62, Repoman-2.3.11

### CODE REVIEW

File: x11-themes/arc-kde/arc-kde-20180614.ebuild [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+DEPEND=""
+RDEPEND=""
+
+pkg_postinst() {
+	elog ""
+	elog "This theme optionally supports \"x11-themes/kvantum\""
+	elog "See also \"x11-themes/arc-theme\" for gtk support"
+	elog ""
+}
```
Comment (gyakovlev) [inline x11-themes/arc-kde/arc-kde-20180614.ebuild]: please add
`RESTRICT="binchecks strip"`
because those checks are not required for this package.

Comment (gcarq) [reply x11-themes/arc-kde/arc-kde-20180614.ebuild]: Added `RESTRICT="binchecks strip"` and force pushed branch

**Comment by a17r** [comment]
This theme is available through store.kde.org (which means you can download it via KDE GHNS or discover). It does not need special permissions for the user to install, so I don't see the reason to provide it as package?

**Comment by gcarq** [comment]
I think it adds some value to have it as package for the following reasons:
* Keeps bootstrapping the system as simple as possible, only a list of packages is needed without pulling bits from different sources
* Update to a newer version even if the KDE Store maintainer is inactive, for example Thermal Monitor on KDE Store is at 1.2.8 while 1.2.9 is out
* Only adds a single copy in multi-user environments

**Pull request closed without merging**